src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -481,3 +481,62 @@ def process_repair(manager, broker, dry_run):
         if pid not in set_process_tasks:
             process_controller.continue_process(pid)
             echo.echo_report(f'Revived process `{pid}`')
+
+
+@verdi_process.command('dump')
+@arguments.PROCESS()
+@options.PATH()
+@options.OVERWRITE()
+@options.INCLUDE_INPUTS()
+@options.INCLUDE_OUTPUTS()
+@options.INCLUDE_ATTRIBUTES()
+@options.INCLUDE_EXTRAS()
+@options.FLAT()
+def process_dump(
+    process,
+    path,
+    include_inputs,
+    include_outputs,
+    include_attributes,
+    include_extras,
+    overwrite,
+    flat,
+) -> None:
+    """Dump process input and output files to disk.
+
+    Child calculations/workflows (also called `CalcJob`s and `WorkChain`s in AiiDA jargon) run by the parent workflow
+    are contained in the directory tree as sub-folders and are sorted by their creation time. The directory tree thus
+    mirrors the logical execution of the workflow, which can also be queried by running `verdi process status <pk>` on
+    the command line.
+
+    By default, input and output files of each calculation can be found in the corresponding "raw_inputs" and
+    "raw_outputs" directories (the former also contains the hidden ".aiida" folder with machine-readable job execution
+    settings). Additional input files (depending on the type of calculation) are placed in the "inputs".
+
+    Lastly, every folder also contains a hidden, human-readable `.aiida_node_metadata.yaml` file with the relevant AiiDA
```

Comment:
```suggestion
    Lastly, every folder also contains a hidden, human-readable `.aiida_node_metadata.yaml` safeguard file with the relevant AiiDA
```
so it is better understandable with the error message

Reply:
Just for clarification, the `.aiida_node_metadata.yaml` file's main purpose is to actually contain relevant metadata for each `ProcessNode` **not** just as a safeguard file for deletion. The reason why it is now also used for the latter purpose is because during early development of the feature, I ended up being in the wrong directory and using the `--overwrite` flag deleted a list of directories which I didn't intend to be deleted. That's why I'm now also checking for this file for safety reasons, to ensure that removing files / directories due to `--overwrite` is only applied if the referenced path was created by running the `dump` command. As this is more of an internal implementation detail, I wouldn't highlight it to the user at this point.